cfalertdialog/src/main/java/com/crowdfireapp/cfalertdialog/CFAlertDialog.java

```diff
@@ -626,16 +951,18 @@ public CFAlertDialog show() {
     private static class DialogParams {
 
         private Context context;
+        private CFAlertBackgroundStyle backgroundStyle = CFAlertBackgroundStyle.PLAIN;
+        private @ColorInt int backgroundColor = -1;
         private CharSequence message, title;
-        private int theme = R.style.CFDialogStyle,
-                dialogGravity = -1,
+        private int theme = R.style.CFDialog,
+                dialogGravity = Gravity.CENTER,
                 textGravity = Gravity.LEFT,
                 iconDrawableId = -1,
                 contentImageDrawableId = -1;
         private View headerView, footerView;
         private int headerViewId = -1, footerViewId = -1;
         private Drawable contentImageDrawable, iconDrawable;
```

Comment:
This is kept so that there is flexibility for the developer who will in turn use this library. Trying to keep in par with the native API as much as possible.
They can pass a programmatically generated view object or any custom layout resource as header/footer, etc.